egui/src/widgets/plot/mod.rs

```diff
@@ -1254,7 +1352,7 @@ impl PreparedPlot {
         }
     }
 
-    fn hover(&self, ui: &Ui, pointer: Pos2, shapes: &mut Vec<Shape>) {
+    fn hover(&self, ui: &Ui, pointer: Pos2, shapes: &mut Vec<Shape>) -> Vec<Cursor> {
```

Comment:
it's a bit unintuitive how hovering can return more than one cursor. I guess it is the horizontal + vertical lines that are considered separate cursors? Could perhaps the code be made simpler by using `struct Cursor { point: PlotPoint, horizontal: bool, vertical: bool }` ?

Reply:
Things like bar charts have too many lines for that to work.

Reply:
Then I don't understand what a `Cursor` is. I thought it was referring to the mouse cursor

Reply:
It's the lines for objects near the mouse cursor or lines for the cursor itself. See the Plot demo -> Charts -> Box Plot and hover over things.